apps/docs/src/app/core/component-docs/fixed-card-layout/examples/default/fixed-card-layout-examples.component.html

```diff
@@ -105,44 +105,44 @@ <h2 fd-card-title>Table card</h2>
                         </thead>
                         <tbody fd-table-body>
                             <tr fd-table-row [main]="true" [activable]="true" tabindex="0">
-                                <td fd-table-cell>
+                                <td fd-table-cell title="Name">
```

Comment:
Titles it's not about the a11y even as they're read by the screen-reader. It's about describing the element when the purpose is not univocal. For example

![image](https://user-images.githubusercontent.com/20265336/135446972-0291d4c3-ac5b-45df-8d87-c4dd5a2af6c6.png)

We all know that the hamburger buttons stand for the menu but there are people who don't know about it so the title gives you a hint.